src/app/api/userAuth.ts

```
@@ -0,0 +1,66 @@
+		const error = await response.json();
+		throw new Error(error.message);
+	}
+
+	const data = await response.json();
+	const token = data.token;
+	// 토큰 만료 시간을 1시간으로 설정 (현재 시간 + 1시간)
+	const expiryTime = Date.now() + 60 * 60 * 1000;
+	// 토큰 저장
+	localStorage.setItem('authToken', token);
+	localStorage.setItem('tokenExpiryTime', expiryTime.toString()); // 만료 시간 저장
+	console.log(data);
```

ESLint

[error] Unexpected console statement.

(no-console)